scripts/travis/conan.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Why sh and not bash? Is there any restrictions to use bash?

Reply:
I tend to use sh, when bash isn't specifically necessary. I've changed it nontheless to be a little more consistent